site/tsconfig.dist.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "extends": "../tsconfig.json",
+    "noEmit": false
```

Comment:
The rationale here is that the site just needs to be type-checked (so don't emit output, i.e don't compile to javascript). The Gatsby config should take care of the rest...